app/views/unattended/provisioning_templates/snippet/redhat_register.erb

```diff
@@ -86,9 +86,10 @@ snippet: true
       subscription_manager_atomic_url = subscription_manager_configuration_url(@host, false)
       subscription_manager_org = @host.rhsm_organization_label
       activation_key = host_param('kt_activation_keys')
-      redhat_install_agent = true
+      redhat_install_agent = host_param_true?('redhat_install_agent')
       redhat_install_host_tools = true
-      redhat_install_host_tracer_tools = false
+      redhat_install_host_tools = false if host_param_false?('redhat_install_host_tools')
+      redhat_install_host_tracer_tools = host_param_true?('redhat_install_host_tracer_tools')
     else
```

Comment:
I think this can be consolidated like so:
```
redhat_install_agent = host_param_true?('redhat_install_agent', true)
redhat_install_host_tools = host_param_true?('redhat_install_host_tools', true)
redhat_install_host_tracer_tools = host_param_true?('redhat_install_host_tracer_tools')
```

I would prefer to continue installing katello-agent by default even though it's deprecated because I think changing the default behavior at this time would cause a bit of a headache for some.